packages/react-components/src/components/VideoTile.test.tsx

```diff
@@ -26,22 +24,33 @@ describe('VideoTile', () => {
     const mockCallback = jest.fn();
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const videoTileProps = { onLongTouch: mockCallback } as any;
-    const wrapper = mount(<VideoTile {...videoTileProps} />);
+    const { container } = render(<VideoTile {...videoTileProps} />);
+    const videoTile = container.querySelector('[data-ui-id="video-tile"]') as HTMLElement;
+    expect(videoTile).toBeTruthy();
+
+    jest.useFakeTimers();
     await act(async () => {
-      wrapper.simulate('touchstart');
-      await new Promise((resolve) => setTimeout(resolve, 1000));
-      wrapper.simulate('touchend');
+      fireEvent.touchStart(videoTile);
+      jest.runAllTimers();
+      fireEvent.touchEnd(videoTile);
```

Comment:
Thanks for this conversion. I was worried about triggering touch events with React Testing Library. Nicely done.